nvflare/private/fed/server/sys_cmd.py

```diff
@@ -32,10 +32,13 @@ def _parse_replies(conn, replies):
         client_name = engine.get_client_name_from_token(r.client_token)
 
         if r.reply:
-            try:
-                resources = json.loads(r.reply.body)
-            except BaseException as e:
-                resources = f"Bad replies: {secure_format_exception(e)}"
+            if isinstance(r.reply.body, str) and r.reply.body:
+                resources = r.reply.body
+            else:
+                try:
+                    resources = json.loads(r.reply.body)
+                except BaseException as e:
+                    resources = f"Bad replies: {secure_format_exception(e)}"
```

Comment:
The new code's behavior is diff. VS the old code.

Consider the case where `r.reply.body = '{"hello": "motor"}' `

The old code resources going to be a dict `{"hello": "motor"}`
The new code resources will be a string `'{"hello": "motor"}' `

Is this what we want?

Reply:
Intended to handle the client authorization error case. Should not simply check the "str" response body type. Changed to use the proper error returnCode.